### ISSUE TITLE
Can't install Salt on Debian Buster arm64

### ISSUE THREAD

**Issue by keenobi** [saltstack/salt-bootstrap#1872]
### Description of Issue/Question

I am trying to install salt on a Debian Buster machine with an arm64 architecture but I get an error when running the script :

```
root@host:~ # ./bootstrap-salt.sh stable 3005
[…]
*  WARN: Support for arm64 packages is experimental and might rely on architecture-independent packages from the amd64 repository.
[…]
* ERROR: https://repo.saltproject.io/py3/debian/10/arm64/3005/salt-archive-keyring.gpg failed to download to /tmp/salt-gpg-Oj5l62UU.pub
* ERROR: Failed to run install_debian_deps()!!!
```
Indeed the arm64 architecture is not supported in the official repo : [https://repo.saltproject.io/py3/debian/10/](https://repo.saltproject.io/py3/debian/10/)

This problem has already been discussed in issue #1240 and a PR (#1251) has been merged.
However, this fix was removed in a recent PR :  #1867.

I thought of several solutions :

- Add support for arm64 architectures
- Test if the architecture is supported for current distribution
  - if yes : Use current architecture
  - if no : Use amd64 architecture by default
- Other ?

### Setup

Debian 10 (arm64) host.

### Steps to Reproduce Issue

Run bootstrap script on Debian 10 with arm64 architecture.

### Versions and Systems

```
root@host:~ # ./bootstrap-salt.sh -v
./bootstrap-salt.sh -- Version 2022.10.04

root@host:~ # cat /etc/debian_version 
10.13

root@host:~ # dpkg --print-architecture
arm64
```

**Comment by garethgreenaway** [comment]
@keenobi I just tested this one and it seems to be working with the latest bootstrap script.

**Comment by keenobi** [comment]
Hello, I don't understand your answer. You tried to install Salt via the bootstrap script on a Debian 10 host with arm64 architecture.

I just tested and it doesn't work :

```
root@host:/tmp # wget https://bootstrap.saltproject.io -O /tmp/bootstrap-salt.sh
root@host:/tmp # /tmp/bootstrap-salt.sh -v
/tmp/bootstrap-salt.sh -- Version 2022.10.04
root@host:/tmp # /tmp/bootstrap-salt.sh stable 3005
 *  INFO: Running version: 2022.10.04
 *  INFO: Executed by: /bin/sh
 *  INFO: Command line: '/tmp/bootstrap-salt.sh stable 3005'

 *  INFO: System Information:
 *  INFO:   CPU:          
 *  INFO:   CPU Arch:     aarch64
 *  INFO:   OS Name:      Linux
 *  INFO:   OS Version:   4.19.0-19-arm64
 *  INFO:   Distribution: Debian 10

 *  INFO: Installing minion
 *  INFO: Found function install_debian_deps
 *  INFO: Found function config_salt
 *  INFO: Found function preseed_master
 *  INFO: Found function install_debian_stable
 *  INFO: Found function install_debian_restart_daemons
 *  INFO: Found function daemons_running
 *  INFO: Found function install_debian_check_services
 *  INFO: Running install_debian_deps()
Hit:1 http://security.debian.org/debian-security buster/updates InRelease
Hit:2 http://cdn-aws.deb.debian.org/debian buster InRelease
Hit:3 http://cdn-aws.deb.debian.org/debian buster-updates InRelease
Hit:4 http://cdn-aws.deb.debian.org/debian buster-backports InRelease
Ign:7 https://repo.saltproject.io/py3/debian/10/arm64/3005 buster InRelease
Err:8 https://repo.saltproject.io/py3/debian/10/arm64/3005 buster Release
  404  Not Found [IP: 13.249.9.66 443]
Reading package lists...
 * ERROR: Failed to run install_debian_deps()!!!
```
The script try to access arm64 package but it doesn't exist in official repository : https://repo.saltproject.io/py3/debian/10/

Please let me know if you need more information.

**Comment by garethgreenaway** [comment]
@keenobi Missed that this was running on Debian 10, misread it as Debian 11.  Thanks!  Will investigate a fix.

**Comment by garethgreenaway** [comment]
Closed by #1887
